source/configure_project/global_configuration/index.html.md.erb

```diff
@@ -283,4 +283,17 @@ source_urls:
   report_issue_url: mailto:support@example.com
 ```
 
+## Build your site using relative links
+
+You can configure your Tech Docs Template (TDT) site to use relative links to pages and assets.
```

Comment:
Given that this is the TDT documentation, do we need to say "your Tech Docs Template (TDT) site" or can we just say "your documentation site"?

Reply:
Good point! Although if we can make this a little more specific, I think it might make sense to keep the TDT mention. For example:

"You can configure the Tech Docs Template to build your site using relative links..."

Reply:
happy to just say that, keeps it cleaner

Reply:
oh - my GH hadn't refreshed yet so missed Mark's comment ... will add that in now